README.md

```diff
@@ -7,11 +7,16 @@ Infer a tree sequence from genetic variation data
 
 The [documentation](https://tskit.dev/tsinfer/docs/stable) contains details of how to use this software, including [installation instructions](https://tskit.dev/tsinfer/docs/stable/installation.html).
 
-The algorithm, its rationale, and results from testing on simulated and real data are described in the following [Nature Genetics paper](https://doi.org/10.1038/s41588-019-0483-y):
+The initial algorithm, its rationale, and results from testing on simulated and real data are described in the following [Nature Genetics paper](https://doi.org/10.1038/s41588-019-0483-y):
 
 > Jerome Kelleher, Yan Wong, Anthony W Wohns, Chaimaa Fadil, Patrick K Albers and Gil McVean (2019) *Inferring whole-genome histories in large population datasets*. Nature Genetics **51**: 1330-1338
 
-Please cite this if you use ``tsinfer`` in your work. Code to reproduce the results in the paper is present in a [separate GitHub repository](https://github.com/mcveanlab/treeseq-inference).
+Tsinfer version 2, which allows missing data and provides a fully parameterised Li & Stephens matching algorithm (i.e. which allows mismatch), is described in the
```

Comment:
"version 2" isn't very accurate and may confuse. Can we be more precise?

Reply:
Done: reworded.